README.md

```diff
@@ -59,6 +61,22 @@ We use Ubuntu 20.04 as the base image for building the systems.
 
 We use [hyperfine](https://github.com/sharkdp/hyperfine) to run the benchmarks.
 
+## Benchmark availability
+
+|        | N-queens           | Generator          | Tree explore       |
```

Comment:
I was thinking we should put systems a long the horizontal axis, and benchmarks along the vertical axis. The rationale being that I envisage far more benchmarks than systems and vertical scroll is more convenient than horizontal scroll.  Let me know your thoughts on this matter.

Reply:
Yes, that makes more sense to me.

Reply:
That rationale seems reasonable. I was thinking of a more Wikipedia like approach (https://en.wikipedia.org/wiki/Dependent_type#Comparison_of_languages_with_dependent_types), but in light of this being a benchmark repot, not a system repo, this makes more sense.

Reply:
In light of #1, we should also envision, that at some point the checks will be converted to badges